application/src/main/java/org/opentripplanner/apis/vectortiles/DebugStyleSpec.java

```diff
@@ -14,6 +13,7 @@
 import org.opentripplanner.apis.vectortiles.model.ZoomDependentNumber;
 import org.opentripplanner.apis.vectortiles.model.ZoomDependentNumber.ZoomStop;
 import org.opentripplanner.service.vehiclerental.street.StreetVehicleRentalLink;
+import org.opentripplanner.standalone.config.debuguiconfig.BackgroundTileLayer;
```

Comment:
config.debuguiconfig seems like a bit of repetition (however the same repetition seems to be done for other configs as well for some reason). Perhaps config.debugui/debugclient)

Reply:
For go or bad, I want to follow the previous naming pattern.

If we can agree to a new structure, I'm happy to make a follow up.